integration_test/src/alph.ts

```diff
@@ -82,7 +82,7 @@ export async function createAlephium(): Promise<AlephiumBridgeChain> {
   const bridgeRewardRouterAddress = addressFromContractId(bridgeRewardRouterId)
   const sequence = new Sequence()
   const defaultArbiterFee = 0n
-  const defaultConfirmations = 1
+  const defaultConfirmations = 0
```

Comment:
Question: why do we need to set the the default confirmation to `0` here?
Could you please remind me on the reasoning behind using timestamp?

Reply:
Thanks a lot for the great explanation :+1: